oaknational/Oak-Web-Application
# [CUR-594] Fix child-subject button ordering in curriculum visualiser

## Description
Fixed child-subject ordering in the curriculum visualiser. Rules are as follows

> Alphabetical but "combined-science" always comes first 

## Issue(s)

`CUR-594`

## How to test

1. Go to https://deploy-preview-2482--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary-aqa/units
2. Check child subjects are in the correct order as per `CUR-594`
